Sources/Amplitude/AMPUtils.m

```diff
@@ -172,12 +172,36 @@ + (NSInteger)barBottomOffset {
 + (CGFloat)statusBarHeight {
     CGSize statusBarSize;
     if (@available(iOS 13.0, *)) {
-        statusBarSize = [[[[AMPUtils getSharedApplication].keyWindow windowScene] statusBarManager] statusBarFrame].size;
+        statusBarSize = [[[[AMPUtils getKeyWindow] windowScene] statusBarManager] statusBarFrame].size;
     } else {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+        // Even with the availability check above, Xcode would still emit a deprecation warning here.
+        // Since there's no way that it could be reached on iOS's >= 13.0
+        // (where `[UIApplication statusBarFrame]` was deprecated), we simply ignore the warning.
         statusBarSize = [[AMPUtils getSharedApplication] statusBarFrame].size;
+#pragma clang diagnostic pop
     }
     return MIN(statusBarSize.width, statusBarSize.height);
 }
+
++ (UIWindow *)getKeyWindow {
+    if (@available(iOS 13.0, *)) {
+        for (UIWindow *window in [[AMPUtils getSharedApplication] windows]) {
+            if (window.isKeyWindow) { return window; }
```

Comment:
Nit: Could you make the logic to be a new line? That's the style we follow. :)

Reply:
Sure thing. 

On a related note, regarding the style you follow, are there rules for whitespaces in these places:
```
if (someCondition) {
                  ^

+ (SomeClass *)someMethod:(SomeType *)someParameter {
 ^                                                 ^
```
?

I've seen some inconsistencies there and would be happy to find+replace the heck out of it.

Reply:
@haoliu-amp Done.

Reply:
Yep, you get the spaces correct! 

We do have a lot inconsistencies in our iOS repo but we're trying to unifying it now when we fix and introducing new stuff.

Reply:
As said, if you'd like, I can make one PR that unifies all these whitespace usages. I'm not too bad at find+replacing and like doing these cleanup chores from time to time :)

Reply:
Sure, that'll be great!

Reply:
I'll see if I can get to it in the next days. Have a nice one ✌️

Reply:
Also, to do this, I'd need info on how you'd like your whitespaces. I'd go about unifying these first:
```
if (someCondition) {
  ^               ^

+ (SomeClass *)someMethod:(SomeType *)someParameter {
 ^          ^                      ^               ^
```

Reply:
This looks awesome to me. Thx!